bokehjs/src/coffee/widget/slider.coffee

```diff
@@ -1,8 +1,9 @@
-_ = require "underscore"
+_ = Bokeh._
 ContinuumView = require "../common/continuum_view"
 HasParent = require "../common/has_parent"
 {logger} = require "../common/logging"
 slidertemplate = require "./slidertemplate"
+require "jquery-ui/slider"
```

Comment:
We'll have to watch out for these. Since we were bringing in jquery-ui in bulk, we were getting lazy with our dependencies